src/diffusers/utils/testing_utils.py

```diff
@@ -252,6 +252,20 @@ def require_torch_2(test_case):
     )
 
 
+def require_torch_version_greater_equal(torch_version):
+    """Decorator marking a test that requires torch with a specific version or greater."""
+
+    def decorator(test_case):
+        correct_torch_version = version.parse(
+            version.parse(torch.__version__).base_version
+        ) >= version.parse(torch_version)
```

Comment:
I think we can just do this?
```python
correct_torch_version = is_torch_available() and is_torch_version(">=", torch_version)
```

cc @BenjaminBossan, want to double check with you because it's different from` require_peft_version_greater` and `require_accelerator_version_greater`, I'm not sure if there is specific reason to do it that way https://github.com/huggingface/diffusers/blob/14a1b86fc7de53ff1dbf803f616cbb16ad530e45/src/diffusers/utils/testing_utils.py#L362

Reply:
went ahead and simplified the decorator in the latest commit, can always revert